datasets/conll2000/conll2000.py

```diff
@@ -0,0 +1,125 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Introduction to the CoNLL-2000 Shared Task: Chunking"""
+
+import logging
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{tksbuchholz2000conll,
+   author     = "Tjong Kim Sang, Erik F. and Sabine Buchholz",
+   title      = "Introduction to the CoNLL-2000 Shared Task: Chunking",
+   editor     = "Claire Cardie and Walter Daelemans and Claire
+                 Nedellec and Tjong Kim Sang, Erik",
+   booktitle  = "Proceedings of CoNLL-2000 and LLL-2000",
+   publisher  = "Lisbon, Portugal",
+   pages      = "127--132",
+   year       = "2000"
+}
+"""
+
+_DESCRIPTION = """\
+ Text chunking consists of dividing a text in syntactically correlated parts of words. For example, the sentence
+ He reckons the current account deficit will narrow to only # 1.8 billion in September . can be divided as follows:
+[NP He ] [VP reckons ] [NP the current account deficit ] [VP will narrow ] [PP to ] [NP only # 1.8 billion ]
+[PP in ] [NP September ] .
+
+Text chunking is an intermediate step towards full parsing. It was the shared task for CoNLL-2000. Training and test
+data for this task is available. This data consists of the same partitions of the Wall Street Journal corpus (WSJ)
+as the widely used data for noun phrase chunking: sections 15-18 as training data (211727 tokens) and section 20 as
+test data (47377 tokens). The annotation of the data has been derived from the WSJ corpus by a program written by
+Sabine Buchholz from Tilburg University, The Netherlands.
+"""
+
+_URL = "https://github.com/teropa/nlp/raw/master/resources/corpora/conll2000/"
+_TRAINING_FILE = "train.txt"
+_TEST_FILE = "test.txt"
+
+
+class Conll2000Config(datasets.BuilderConfig):
+    """BuilderConfig for Conll2000"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig forConll2000.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(Conll2000Config, self).__init__(**kwargs)
+
+
+class Conll2000(datasets.GeneratorBasedBuilder):
+    """Conll2000 dataset."""
+
+    BUILDER_CONFIGS = [
+        Conll2000Config(name="conll2000", version=datasets.Version("1.0.0"), description="Conll2000 dataset"),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "word": datasets.Sequence(datasets.Value("string")),
```

Comment:
Could you change `word` to `words`, for consistency with the other datasets ?

Reply:
Done, but spread out in e711cee and 3b860b4 Forgot to rerun dataset_infos.json generation again.